assets/export/docx/7x10/word/styles.xml

```diff
@@ -0,0 +1,1952 @@
+        </w:pPr>
+        <w:rPr>
+            <w:sz w:val="22"/>
+        </w:rPr>
+    </w:style>
+    <w:style w:type="paragraph" w:styleId="Heading4">
+        <w:name w:val="heading 4"/>
+        <w:basedOn w:val="Heading3"/>
+        <w:next w:val="BodyText"/>
+        <w:uiPriority w:val="9"/>
+        <w:unhideWhenUsed/>
+        <w:uiPriority w:val="2"/>
+        <w:qFormat/>
+        <w:pPr>
+            <w:outlineLvl w:val="3"/>
+        </w:pPr>
+        <w:rPr>
+            <w:sz w:val="18"/>
+            <w:u w:val="single"/>
+        </w:rPr>
+    </w:style>
+    <w:style w:type="paragraph" w:styleId="Heading5">
+        <w:name w:val="heading 5"/>
+        <w:basedOn w:val="Heading4"/>
+        <w:next w:val="BodyText"/>
+        <w:uiPriority w:val="9"/>
+        <w:unhideWhenUsed/>
+        <w:uiPriority w:val="2"/>
+        <w:qFormat/>
+        <w:pPr>
+            <w:spacing w:before="360" w:after="180"/>
+            <w:outlineLvl w:val="4"/>
+        </w:pPr>
+        <w:rPr>
+            <w:b w:val="0"/>
+        </w:rPr>
+    </w:style>
+    <w:style w:type="paragraph" w:styleId="Heading6">
+        <w:name w:val="heading 6"/>
+        <w:basedOn w:val="Heading5"/>
+        <w:next w:val="BodyText"/>
+        <w:uiPriority w:val="9"/>
+        <w:unhideWhenUsed/>
+        <w:uiPriority w:val="2"/>
+        <w:qFormat/>
+        <w:pPr>
+            <w:outlineLvl w:val="5"/>
+        </w:pPr>
+        <w:rPr>
+            <w:i/>
+            <w:u w:val="none"/>
+        </w:rPr>
+    </w:style>
+    <w:style w:type="paragraph" w:styleId="Heading7">
+        <w:name w:val="heading 7"/>
+        <w:basedOn w:val="Heading6"/>
+        <w:next w:val="BodyText"/>
+        <w:uiPriority w:val="9"/>
+        <w:unhideWhenUsed/>
+        <w:uiPriority w:val="2"/>
+        <w:qFormat/>
+        <w:pPr>
+            <w:outlineLvl w:val="6"/>
+        </w:pPr>
+    </w:style>
+    <w:style w:type="paragraph" w:styleId="Heading8">
+        <w:name w:val="heading 8"/>
+        <w:basedOn w:val="Heading7"/>
+        <w:next w:val="BodyText"/>
+        <w:uiPriority w:val="9"/>
+        <w:unhideWhenUsed/>
+        <w:uiPriority w:val="2"/>
+        <w:qFormat/>
+        <w:pPr>
+            <w:outlineLvl w:val="7"/>
+        </w:pPr>
+    </w:style>
+    <w:style w:type="paragraph" w:styleId="Heading9">
+        <w:name w:val="heading 9"/>
+        <w:basedOn w:val="Heading8"/>
+        <w:next w:val="BodyText"/>
+        <w:uiPriority w:val="9"/>
+        <w:unhideWhenUsed/>
+        <w:uiPriority w:val="2"/>
+        <w:qFormat/>
+        <w:pPr>
+            <w:outlineLvl w:val="8"/>
+        </w:pPr>
+    </w:style>
+
+    <!--  The corresponding subheadings are largely the same size, but de-emphasized through color and text style. I don't
+        think these can be implemented in the h2o interface, but users might find them useful. -->
+    <w:style w:type="paragraph" w:customStyle="1" w:styleId="Subheading1">
```

Comment:
Can you expand on "I don't think these can be implemented in the h2o interface"?

Reply:
I don't see a way to set any text as subheading in the casebook editor.

Reply:
Oh, gotcha, thx!